Dockerfile

```diff
@@ -16,27 +16,22 @@ COPY ./trin-core ./trin-core
 COPY ./trin-history ./trin-history 
 COPY ./trin-state ./trin-state 
 COPY ./ethportal-peertest ./ethportal-peertest 
-COPY ./mainnetMM ./mainnetMM 
-COPY ./utp-testing ./utp-testing 
+COPY ./utp-testing ./utp-testing
 
 # build for release
 RUN cargo build --all --release
 
 # final base
-FROM rust:1.62
+FROM ubuntu:22.04
 
-# copy build artifact from build stage
-COPY --from=builder /trin/target/release/trin .
+# copy build artifacts from build stage
+# rename trin executable to enable mkdir /trin
+COPY --from=builder /trin/target/release/trin ./trin-main
 COPY --from=builder /trin/target/release/trin-cli .
-COPY --from=builder /trin/target/release/seed-database .
-
-# These steps copy the mainnet chain header seed data into container
-# This data is too large to be kept inside trin-source code
-# It must be downloaded separately and moved to the correct location
-# https://www.dropbox.com/s/y5n36ztppltgs7x/mainnetMM.zip?dl=0
-RUN mkdir /mainnetMM
-COPY --from=builder /trin/mainnetMM ./mainnetMM
+# copy default merge master acc to expected location
+RUN mkdir -p /trin/trin-core/src/assets
+COPY --from=builder /trin/trin-core/src/assets/merge_macc.bin ./trin/trin-core/src/assets/merge_macc.bin
 
 ENV RUST_LOG=debug
 
-ENTRYPOINT ["./trin"]
+ENTRYPOINT ["./trin-main"]
```

Comment:
Could we just run line `32` before line `29` and keep the executable named `trin`?